plugins/cloud-hypervisor/internal/handlers/dump_handlers.go

```diff
@@ -0,0 +1,57 @@
+package handlers
+
+import (
+	"context"
+
+	"buf.build/gen/go/cedana/cedana/protocolbuffers/go/daemon"
+	"github.com/cedana/cedana/pkg/types"
+	"github.com/cedana/cedana/pkg/utils"
+	clh "github.com/cedana/cedana/plugins/cloud-hypervisor/pkg/clh"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var Dump types.DumpVM = dump
+
+type VMSnapshotter interface {
```

Comment:
I’d put this somewhere in `pkg`, with CLH being it’s implementation. `pkg/vm/snaphotter.go` can container just the interface. `pkg/vm/snapshotter_clh.go` contains the CLH implementation. Since it’d be in a package called `vm`, the interface can simply be called `Snapshotter`, so that wherever it’s imported, it’s imported as `vm.Snapshotter`, for simplicity.